hack/testing/run_daemon_cri_integration.sh

```diff
@@ -127,14 +137,14 @@ integration::run_daemon_cri_test_cases() {
   sleep 5
 }
 
-main() {
-  local cmd flags
+integration::run_cri_test(){
+  local cri_runtime cmd flags
+  cri_runtime=$1
   cmd="pouchd-integration"
   flags=" -test.coverprofile=${coverage_profile} DEVEL"
```

Comment:
we should use different coverage-profile for alpha1 otherwise the alpha2 will override the result.